### PR TITLE
Prepare Iris test release notes

### PR DESCRIPTION
**Pull request by dincho**

### CODE REVIEW

**Comment by dincho** [comment]
I intentionally kept the release notes in the next and next-iris dirs.

**Review by velzevur — DISMISSED** [review]

**Review by uwiger — APPROVED** [review]

File: docs/release-notes/RELEASE-NOTES-0.0.0-iris.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# About this release
+
+[This](https://github.com/aeternity/aeternity/releases/tag/v0.0.0-iris) is a Iris test release.
```
Comment (cytadela8) [inline docs/release-notes/RELEASE-NOTES-0.0.0-iris.md:3]: Is this tag correct?

File: docs/release-notes/RELEASE-NOTES-0.0.0-iris.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      "1": 0
+      "2": 1
+      "3": 2
+      "4": 3
+
+ fork_management:
+    network_id: "my_test_iris"
+```
+
+Please et us know if you have any problems by [opening a ticket](https://github.com/aeternity/aeternity/issues).
```
Comment (cytadela8) [inline docs/release-notes/RELEASE-NOTES-0.0.0-iris.md:72]:
```suggestion
Please let us know if you have any problems by [opening a ticket](https://github.com/aeternity/aeternity/issues).
```

File: docs/release-notes/RELEASE-NOTES-0.0.0-iris.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+# About this release
+
+[This](https://github.com/aeternity/aeternity/releases/tag/v0.0.0-iris) is a Iris test release.
+
+It:
```
Comment (cytadela8) [inline docs/release-notes/RELEASE-NOTES-0.0.0-iris.md:5]: "It:"?  Maybe "Changes:"

File: docs/release-notes/RELEASE-NOTES-0.0.0-iris.md (hunk 1 above)
Comment (dincho) [reply docs/release-notes/RELEASE-NOTES-0.0.0-iris.md:3]: in what context ? it is semver compatible

File: docs/release-notes/RELEASE-NOTES-0.0.0-iris.md (hunk 3 above)
Comment (dincho) [reply docs/release-notes/RELEASE-NOTES-0.0.0-iris.md:5]: this is kind of template we use for long time

File: docs/release-notes/RELEASE-NOTES-0.0.0-iris.md (hunk 1 above)
Comment (cytadela8) [reply docs/release-notes/RELEASE-NOTES-0.0.0-iris.md:3]: Just making sure "v0.0.0" is what we want.

File: docs/release-notes/RELEASE-NOTES-0.0.0-iris.md (hunk 3 above)
Comment (cytadela8) [reply docs/release-notes/RELEASE-NOTES-0.0.0-iris.md:5]: Ok, I guess it's fine.

**Review by gorbak25 — APPROVED** [review]

**Review by uwiger — APPROVED** [review]

**Review by cytadela8 — APPROVED** [review]

**Review by velzevur — APPROVED** [review]

**Pull request merged**